app/src/androidTest/java/com/cmput301w23t09/qrhunter/BaseTest.java

```diff
@@ -0,0 +1,77 @@
+package com.cmput301w23t09.qrhunter;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import com.cmput301w23t09.qrhunter.database.DatabaseConnection;
+import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.QuerySnapshot;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import org.junit.After;
+import org.junit.BeforeClass;
+
+/** Base class that implements destruction of the database after each test is completed. */
+public abstract class BaseTest {
+
+  private static final String COLLECTION_PREFIX = "test_";
```

Comment:
When the collection is destroyed, will it still show up as an empty collection in Firebase or will it just be gone? If not, we should add a random number in the collection prefix so that we can run tests concurrently without messing with another test run's database.